### ISSUE TITLE
This is bug?

### ISSUE THREAD

**Issue by zhiephie** [barteksc/AndroidPdfViewer#368]
```
java.lang.ArithmeticException: 
 
  at com.github.masalah.pdfviewer.PDFView.g (Unknown Source)
 
  at com.github.masalah.pdfviewer.e.onScroll (Unknown Source)
 
  at android.view.GestureDetector.onTouchEvent (GestureDetector.java:567)
 
  at com.github.masalah.pdfviewer.e.onTouch (Unknown Source)
 
  at android.view.View.dispatchTouchEvent (View.java:8466)
 
  at android.view.ViewGroup.dispatchTransformedTouchEvent (ViewGroup.java:2415)
 
  at android.view.ViewGroup.dispatchTouchEvent (ViewGroup.java:2098)
 
  at android.view.ViewGroup.dispatchTransformedTouchEvent (ViewGroup.java:2421)
 
  at android.view.ViewGroup.dispatchTouchEvent (ViewGroup.java:2112)
 
  at android.view.ViewGroup.dispatchTransformedTouchEvent (ViewGroup.java:2421)
 
  at android.view.ViewGroup.dispatchTouchEvent (ViewGroup.java:2112)
 
  at android.view.ViewGroup.dispatchTransformedTouchEvent (ViewGroup.java:2443)
 
  at android.view.ViewGroup.dispatchTouchEvent (ViewGroup.java:2112)
 
  at android.view.ViewGroup.dispatchTransformedTouchEvent (ViewGroup.java:2421)
 
  at android.view.ViewGroup.dispatchTouchEvent (ViewGroup.java:2112)
 
  at android.view.ViewGroup.dispatchTransformedTouchEvent (ViewGroup.java:2421)
 
  at android.view.ViewGroup.dispatchTouchEvent (ViewGroup.java:2112)
 
  at android.view.ViewGroup.dispatchTransformedTouchEvent (ViewGroup.java:2421)
 
  at android.view.ViewGroup.dispatchTouchEvent (ViewGroup.java:2112)
 
  at com.android.internal.policy.impl.PhoneWindow$DecorView.superDispatchTouchEvent (PhoneWindow.java:2406)
 
  at com.android.internal.policy.impl.PhoneWindow.superDispatchTouchEvent (PhoneWindow.java:1731)
 
  at android.app.Activity.dispatchTouchEvent (Activity.java:2785)
 
  at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent (Unknown Source)
 
  at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent (PhoneWindow.java:2367)
 
  at android.view.View.dispatchPointerEvent (View.java:8667)
 
  at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent (ViewRootImpl.java:4155)
 
  at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess (ViewRootImpl.java:4006)
 
  at android.view.ViewRootImpl$InputStage.deliver (ViewRootImpl.java:3561)
 
  at android.view.ViewRootImpl$InputStage.onDeliverToNext (ViewRootImpl.java:3614)
 
  at android.view.ViewRootImpl$InputStage.forward (ViewRootImpl.java:3580)
 
  at android.view.ViewRootImpl$AsyncInputStage.forward (ViewRootImpl.java:3697)
 
  at android.view.ViewRootImpl$InputStage.apply (ViewRootImpl.java:3588)
 
  at android.view.ViewRootImpl$AsyncInputStage.apply (ViewRootImpl.java:3754)
 
  at android.view.ViewRootImpl$InputStage.deliver (ViewRootImpl.java:3561)
 
  at android.view.ViewRootImpl$InputStage.onDeliverToNext (ViewRootImpl.java:3614)
 
  at android.view.ViewRootImpl$InputStage.forward (ViewRootImpl.java:3580)
 
  at android.view.ViewRootImpl$InputStage.apply (ViewRootImpl.java:3588)
 
  at android.view.ViewRootImpl$InputStage.deliver (ViewRootImpl.java:3561)
 
  at android.view.ViewRootImpl.deliverInputEvent (ViewRootImpl.java:5839)
 
  at android.view.ViewRootImpl.doProcessInputEvents (ViewRootImpl.java:5813)
 
  at android.view.ViewRootImpl.enqueueInputEvent (ViewRootImpl.java:5784)
 
  at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent (ViewRootImpl.java:5929)
 
  at android.view.InputEventReceiver.dispatchInputEvent (InputEventReceiver.java:185)
 
  at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents (Native Method)
 
  at android.view.InputEventReceiver.consumeBatchedInputEvents (InputEventReceiver.java:176)
 
  at android.view.ViewRootImpl.doConsumeBatchedInput (ViewRootImpl.java:5900)
 
  at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run (ViewRootImpl.java:5952)
 
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:773)
 
  at android.view.Choreographer.doCallbacks (Choreographer.java:586)
 
  at android.view.Choreographer.doFrame (Choreographer.java:554)
 
  at android.view.Choreographer$FrameDisplayEventReceiver.run (Choreographer.java:759)
 
  at android.os.Handler.handleCallback (Handler.java:739)
 
  at android.os.Handler.dispatchMessage (Handler.java:95)
 
  at android.os.Looper.loop (Looper.java:135)
 
  at android.app.ActivityThread.main (ActivityThread.java:5446)
 
  at java.lang.reflect.Method.invoke (Native Method)
 
  at java.lang.reflect.Method.invoke (Method.java:372)
 
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:941)
 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:736)
```

**Comment by barteksc** [comment]
What is `com.github.masalah.pdfviewer.PDFView`?

**Comment by zhiephie** [comment]
This source same with this sample source, but only refactor package name and auto refactor inside a folder in android-pdf-viewer. See this [https://prnt.sc/g0lvwg](https://prnt.sc/g0lvwg).

I don't know why the package name is refactored in the folder android-pdf-viewer.

**Comment by jimmytao** [comment]
i met this problem  as same,the error is divide by zero and i found the reason is my pdf contain password;

**Comment by barteksc** [comment]
Probably fixed in 2.7.0
